README.md

```diff
@@ -55,6 +55,30 @@ And just run:
 jsonschema-gentype
 ```
 
+# Validate
+
+It also provide some validate improvements from `jsonschema` to validate a YAML file.
+
+Enancemments:
```

Comment:
Enancemments => Enhancements
But IMHO the term is not appropriated, maybe:
```
Features:
```

Reply:
=> Additional features